test/controllers/manage/reports_controller_test.rb

```diff
@@ -2,8 +2,17 @@
 
 class Manage::ReportsControllerTest < ActionController::TestCase
   test "heard_about_us" do
```

Comment:
Minor nitpick: You should probably rename this test something like "report endpoints" or something